tests/integration-tests/tests/arm_pl/test_arm_pl/test_arm_pl/pcluster.config.ini

```diff
@@ -0,0 +1,22 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = 0
+max_queue_size = 2
```

Comment:
Why specify max_queue_size? If you're not going to need a compute node I would remove this probably. Minor point though.

Reply:
Removed